### PR TITLE
Mattcrooke navbar2

### PR DESCRIPTION
**Pull request by matthewcrooke1**

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -10,6 +10,16 @@
...
+          = link_to 'Logout', logout_path, {id: 'logo'}
+  
+        - else
+          = link_to "Log in", login_path, {id: 'logo'}
+          = link_to "Sign up", signup_path, {id: 'logo'}
+        
+        
     #body
       = yield
+
```
Comment (GoodOlChuck) [inline app/views/layouts/application.html.haml:21]: You need to add a newline at the end of this file

File: app/assets/stylesheets/application.css.scss [hunk 2]
```diff
@@ -15,24 +15,36 @@ header {
...
   }
-  
+
   a:hover {
     background: hsla(0, 0, 100, 0.2);
   }
 }
 
+
+
```
Comment (GoodOlChuck) [inline app/assets/stylesheets/application.css.scss]: Clean up the extra spaces

File: app/assets/stylesheets/application.css.scss [hunk 3]
```diff
@@ -15,24 +15,36 @@ header {
   flex-flow: row;
   background: purple;
   color: white;
-  
+
+  #navBar {
```
Comment (kevincox) [inline app/assets/stylesheets/application.css.scss]: Is this element in the markup?

File: app/assets/stylesheets/application.css.scss [hunk 4]
```diff
@@ -15,24 +15,36 @@ header {
...
+    display: block;
+    padding: 0.4em;
+    float: left;
+  }
+
   #logo {
     display: block;
     padding: 0.4em;
+    float: left;
```
Comment (kevincox) [inline app/assets/stylesheets/application.css.scss]: Don't use float inside a flexbox. You might need to add an expandable spacer between the right and left. Something like `.spacer { flex: 1 }`.

File: app/views/layouts/application.html.haml [hunk 5]
```diff
@@ -10,6 +10,16 @@
     %header
       #headerbar
         = link_to "Social Wiki", "/", {id: 'logo'}
-    
+        
+        - if current_user
+          = label_tag current_user.username
+          = link_to 'Logout', logout_path, {id: 'logo'}
+
```
Comment (kevincox) [inline app/views/layouts/application.html.haml]: I find this extra line quite confusing. Also there are a lot of extra lines in this file. Sorry for being nitpicky.

**Comment by kevincox** [comment]
:boat: :dash:

**Pull request merged**